tests/adapters/test_http.py

```diff
@@ -1,75 +0,0 @@
-import pytest
```

Comment:
Unit tests for the `HTTPAdapter` would be good

Reply:
I think this will all be changed when we refactor ComposedAdapter, so can wait til then

Reply:
Added to #31